website/source/guides/advanced-scheduling/affinity.html.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: "guides"
+page_title: "Affinity"
+sidebar_current: "guides-advanced-scheduling"
+description: |-
+   The affinity stanza allows operators to express placement preference for a set of nodes. Affinities may be expressed on attributes or client metadata.  
```

Comment:
Shouldn;t this be a description of the guide itself? Something like , the following guide walks through using the affinity stanza in Nomad